<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Registro de datos</string>
      </property>
      <widget class="QSpinBox" name="blue_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>39</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>49</y>
         <width>66</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen en x : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>179</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Color (RGB) </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>153</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Velocidad : </string>
       </property>
      </widget>
      <widget class="QSpinBox" name="origen_y_spinBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>75</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>127</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino y : </string>
       </property>
      </widget>
      <widget class="QSpinBox" name="origen_x_spinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>49</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>101</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino x : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="velocidad_lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>153</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>224</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>green</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="green_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>224</y>
         <width>39</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="destino_x_spinBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>101</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>75</y>
         <width>66</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen en y : </string>
       </property>
      </widget>
      <widget class="QSpinBox" name="destino_y_spinBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>127</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>198</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>red</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>23</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>20</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>blue</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="red_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>198</y>
         <width>39</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QPushButton" name="agregar_final_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar al final</string>
       </property>
      </widget>
      <widget class="QPushButton" name="agregar_inicio_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar al inicio</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mostrar_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mostrar</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="salida">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>261</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_abrir"/>
    <addaction name="action_guardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_abrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_guardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
